conmon-rs/server/src/child_reaper.rs

```diff
@@ -204,15 +205,20 @@ impl ReapableChild {
 
     fn watch(&self) -> Result<(Sender<ExitChannelData>, Receiver<ExitChannelData>)> {
         let exit_paths = self.exit_paths().clone();
+        let oom_exit_paths = self.exit_paths().clone();
```

Comment:
I think we should have a separate spot instead of reusing the exit code paths

Reply:
fixed in the latest push.